schedule/functional/extra_tests_textmode_phub.yaml

```diff
@@ -9,7 +9,9 @@ schedule:
     - '{{wpa_supplicant}}'
     - console/vmstat
     - console/libjpeg_turbo
+    - console/libgit2
     - console/ansible
+    - network/cifs
```

Comment:
Is it the same reason as libgit2?

Reply:
Yes, cifs need to install 'samba-client' which requires PackageHub

Reply:
Then I would suggest changing the git messages and title :)

Reply:
Also this failed job: https://openqa.suse.de/tests/15534070#step/prepare_systemd_and_testsuite/56
I moved the 'prepare_systemd_and_testsuite' to extra_tests_textmode_phub job